jazz-ui/src/app/pages/environment-assets/env-assets-section.component.scss

```diff
@@ -0,0 +1,308 @@
+@import './../../../main-styles';
+.full-page{
+    border-radius: 3px;
+    background-color: #ffffff;
+    box-shadow: 0 2px 4px 0 rgba(0, 0, 0, 0.11);
+
```

Comment:
There are lot of extra lines in multiple places. Is there any reason for that?

Reply:
no reason, might have deleted commented lines from between the code which left empty new lines